wp-content/themes/core/assets/css/src/theme/layouts/default.pcss

```diff
@@ -61,7 +61,7 @@ main {
 
 .l-sink {
 
-	& > *:not(.alignwide):not(.alignfull):not(.alignleft):not(.alignright):not(.is-style-wide) {
+	& > *:not(.alignwide):not(.alignfull):not(.alignleft):not(.alignright):not(.is-style-wide):not(.c-panel) {
```

Comment:
Here and below I'm updating the typographic & spacing sinks to better work with panels, now that we have those available to implement.

Reply:
oof thats a selector. you aint busting that specificity with anything but a nuke

Reply:
:joy: Welcome to selector soup, my fiends.